inc/meta-box-models.php

```diff
@@ -303,6 +303,11 @@ public function validate_taxonomyselect($field, $post_ID) {
  */
 
 public function validate_date($field, $post_ID) {
+    $i = 0;
+    while ( isset( $_POST['rm_' . $field['taxonomy'] . '_' . $i ]) ) {
```

Comment:
Wouldn't this loop logic fail if someone tries to remove the second or later term in the list, but not the first?